pkg/client/model_address.go

```diff
@@ -9,11 +9,12 @@
 
 package client
 
-// CustomerAddress struct for CustomerAddress
-type CustomerAddress struct {
+// Address struct for Address
+type Address struct {
```

Comment:
Was this generated by running `make client`? Or was it manually edited. I ask because of this error in CI:

```
pkg/customers/customer_search.go:243:89: CustomerAddress not declared by package client (compile)
```

Reply:
@adamdecaf I have a separate PR with all of the changes.  I was following your recommendation to keep only the model changes in one PR so those things could be easily viewed and coordinated with Frontend.  I wasn't sure exactly how to do that.  This PR will eventually be closed though.